app/src/main/res/layout/fragment_onboarding.xml

```diff
@@ -0,0 +1,27 @@
+<?xml version="1.0" encoding="utf-8"?>
+<LinearLayout xmlns:android="http://schemas.android.com/apk/res/android"
+    android:layout_width="match_parent"
+    android:layout_height="match_parent"
+    android:layout_margin="4dp"
+    android:background="@color/whiteBackground"
+    android:orientation="vertical">
+
+    <TextView xmlns:android="http://schemas.android.com/apk/res/android"
+        android:layout_width="match_parent"
+        android:layout_height="wrap_content"
+        android:background="@drawable/choose_section_border"
+        android:gravity="center_horizontal"
+        android:text="@string/choose_section"
+        android:textColor="@color/blackText"
+        android:textSize="20sp"
+        android:textStyle="bold" />
```

Comment:
Take a look at `TextAppearance.AppCompat.*` from `AppCompat` library. I think one of these should suite you. Full list https://gist.github.com/Aracem/922d460b73168676ca21